event/manager.go

```diff
@@ -168,13 +168,13 @@ func SaveEventToEtcd(event *api.Event) error {
 	if nil != err {
 		return err
 	}
-	return ec.Set(EventsUnfinished+"/"+string(event.EventID), string(bEvent))
+	return ec.Set(EventsUnfinished+string(event.EventID), string(bEvent))
```

Comment:
go tools auto format it for me, I can't change it
In this case, there are two params in function, each param should be tight I think😁